File: ayt-01/02GCD.c
#include <stdio.h>
int gcd(int m, int n)
{
    printf("%i %i\n", m, n);
    return (m < n) ? gcd(n, m) : (m % n == 0) ? n : gcd(n, m % n);
}
int main(int argv, char **argc)
{
    gcd(9, 4);
    return 0;
}

File: ayt-01/02GCD.java
class gcd
{ 
    static int gcd(int a, int b)
    {
      System.out.println(a + " " + b);
      if(b == 0) return a;
      return gcd(b, a % b);
    }
    public static void main(String args[]) 
    { 
        gcd(16, 8);
    } 
} 